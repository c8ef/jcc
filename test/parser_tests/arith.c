int main() {
  int x = 0;
  x++;
  int y = 0;
  y--;
}
