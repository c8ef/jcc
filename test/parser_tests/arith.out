FunctionDecl: main
  Args[0]
  CompoundStatement
    DeclStatement
      VarDecl: x
        IntergerLiteral: 0
    ExprStatement
      UnaryExpr(++):
        DeclRefExpr: x
    DeclStatement
      VarDecl: y
        IntergerLiteral: 0
    ExprStatement
      UnaryExpr(--):
        DeclRefExpr: y
